### PR TITLE
Move units to attributes and add volume units

### PR DESCRIPTION
**Pull request by Abyss777**
Web part for https://github.com/tananaev/traccar/pull/3507

Some comments in code

### CODE REVIEW

File: web/l10n/en.json [hunk 1]
```diff
@@ -64,6 +68,7 @@
     "sharedTypeNumber": "Number",
     "sharedTypeBoolean": "Boolean",
     "sharedTimezone": "Timezone",
+    "sharedVolumeUnit": "Volume",
```
Comment (Abyss777) [inline web/l10n/en.json]: May be we should add "Unit" word to strings, they look not clearly
image

File: web/app/view/dialog/AttributeController.js [hunk 2]
```diff
@@ -53,35 +53,72 @@ Ext.define('Traccar.view.dialog.AttributeController', {
...
+                            default:
+                                break;
+                        }
+                    } else {
+                        config.xtype = 'textfield';
+                    }
+                    break;
             }
-            if (valueField.getXType() !== config.xtype || valueField.convert !== config.convert) {
+            if (valueField.getXType() !== config.xtype) {
```
Comment (Abyss777) [inline web/app/view/dialog/AttributeController.js]: Handle switching type more correctly here and next

File: web/app/store/VolumeUnits.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+Ext.define('Traccar.store.VolumeUnits', {
+    extend: 'Ext.data.Store',
+    fields: ['key', 'name', 'fullName', 'factor'],
```
Comment (Abyss777) [inline web/app/store/VolumeUnits.js]: Added `fullName` fields to distinguish gallons
image

May be we should do the same for distance and speed units

File: web/l10n/en.json (hunk 1 above)
Comment (tananaev) [reply web/l10n/en.json]: Agree.

**Comment by Abyss777** [comment]
Added "Unit" to strings. Chose "settings" prefix to save one string from translation.
image

Looks better now

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**